src/app/features/dashboard-widgets/entity-count-dashboard-widget/entity-count-dashboard/entity-count-dashboard.component.html

```diff
@@ -66,31 +58,30 @@
     >
       <ng-container matColumnDef="label">
         <td *matCellDef="let group">
-          @if (!group.groupedByEntity) {
-            @if (group.isInvalidOption) {
-              <span
-                class="invalid-label"
-                matTooltip="records with a value that is not matching any available dropdown option"
-                i18n-matTooltip
-                i18n
-              >
-                [invalid option]
-              </span>
-            } @else if (group.label === undefined) {
-              <span
-                class="not-defined-label"
-                matTooltip="records where this field is empty"
-                i18n-matTooltip
-                i18n
-                >not defined</span
-              >
-            } @else {
-              <app-display-configurable-enum
-                [value]="group"
-              ></app-display-configurable-enum>
-            }
-          } @else {
+          @if (group.isInvalidOption) {
+            <span
+              class="invalid-label"
+              matTooltip="records with a value that is not matching any available dropdown option"
+              i18n-matTooltip
+              i18n
+            >
+              [invalid option]
+            </span>
+          } @else if (group.label === undefined) {
+            <span
+              class="not-defined-label"
+              matTooltip="records where this field is empty"
+              i18n-matTooltip
+              i18n
+              >not defined</span
+            >
+          } @else if (group.groupedByEntity) {
             <app-entity-block [entityId]="group.id"></app-entity-block>
```

Comment:
do we still need this separately from the app-entity-field-view mode?

Reply:
It was working fine without it, so I’m removing this for now as well